File: Baek/solved/Baek11723.js
let input = `26
add 1
add 2
check 1
check 2
check 3
remove 2
check 1
check 2
toggle 3
check 1
check 2
check 3
check 4
all
check 10
check 20
toggle 10
remove 20
check 10
check 20
empty
check 1
toggle 1
check 1
toggle 1
check 1`.split("\n").map(e => e.split(" "));
input.shift();

let set = new Array(20).fill(0);
const func = {
    add   : (n) => set[n] = 1,
    remove: (n) => set[n] = 0,
    check : (n) => ans += set[n]+"\n",
    toggle: (n) => set[n]^= 1,
    all   : ( ) => set.fill(1),
    empty : ( ) => set.fill(0)
};
let ans = "";

for (let i = 0, l = input.length; i < l; i++) func[input[i][0]](input[i][1]-1);
console.log(ans.trim());

File: Baek/solved/Baek11723.py
import sys

n = int(sys.stdin.readline())
s = 0
for _ in range(n):
  line = sys.stdin.readline().split()
  command = line[0]
  mask = 0 if len(line) == 1 else (1 << int(line[1]))
  if   command == "add":
    s |= mask
  elif command == "remove":
    s &= ~mask
  elif command == "check":
    print(1 if (s & mask) else 0)
  elif command == "toggle":
    s ^= mask
  elif command == "all":
    s = (1 << 21) - 1
  else:
    s = 0
